packages/testcontainers/src/wait-strategies/host-port-wait-strategy.ts

```diff
@@ -36,22 +42,55 @@ export class HostPortWaitStrategy extends AbstractWaitStrategy {
     boundPorts: BoundPorts
   ): Promise<void> {
     for (const [internalPort] of boundPorts.iterator()) {
+      if (typeof internalPort === "string" && internalPort.toLowerCase().includes("/udp")) {
```

Comment:
`internalPort` is always a `string`

```suggestion
      if (internalPort.toLowerCase().includes("/udp")) {
```